core/tests/widgets/test_container_debug_mode.py

```diff
@@ -0,0 +1,57 @@
+from os import environ
+
+from travertino.colors import rgb
+
+import toga
+
+
+# test that a non-container-like widget in layout debug mode has a default background
+def test_button_debug_background():
+    """A Button in layout debug mode has a default background."""
+    # Enable layout debug mode
+    environ["TOGA_DEBUG_LAYOUT"] = "1"
```

Comment:
This modifies the global environment, so it will leak into external tests (which means this test can impact on other tests). For something like this, it's a good idea to use [`monkeypatch.setenv`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.MonkeyPatch.setenv) - that will automatically reset the environment variable when the test finishes.